src/hooks/use-render.ts

```diff
@@ -31,11 +31,13 @@ const useRender = () => {
   const model = useModel() as Model;
   const app = useApp();
   const embed = useEmbed();
+  const keyboard = useKeyboard();
   const interactions = useInteractionState();
   const [effectiveProperties, isLoadingEffectiveProperties] = useEffectiveProperties(model, layout);
   const layoutService = useLayoutService(layout, effectiveProperties);
   const selections = useSelections() as ExtendedSelections;
-  const theme = useTheme() as ExtendedTheme;
+  const theme = useExtendedTheme(element);
+  const themeName = theme.name();
```

Comment:
Yes, wanted to keep the lint rule. Otherwise the linter will complain that `theme` is missing in the dependency, because you have to use `theme.name()` to catch changes